bases/computer_lib.py

```diff
@@ -216,7 +217,16 @@ def change_text(self, button_index):
                 ShowProgress.activateProgressScreen('loading',3)
                 import dj_lib
                 dj_lib.enable()
-                VS.loadGame(self.picker_screen.items[self.picker_screen.selection].data)
+                game = self.picker_screen.items[self.picker_screen.selection].data
+                if not is_utf8(game):
+                    alert = f'  {game} is not UTF-8, convert it before loading'
+                    print(alert)
+                    self.alert_text = GUI.GUIStaticText(self.guiroom, 'txt_screen', alert,
+                        GUI.GUIRect(60, 90, 657, 100), color=GUI.GUIColor(255, 255, 255), bgcolor=GUI.GUIColor(0, 0, 0))
```

Comment:
We'll need to test this with different game resolutions / window sizes, to make sure it works as desired

Reply:
Indeed; for the record, these values are based on a 1600x900 window

Reply:
yes, if we could make this dynamic for placement and test at various resolutions / window sizes then this should be good. We'll have to make ports to PWCU (for @ermo ).

Reply:
With these values,
```
GUI.GUIRect(40, 60, 438, 100, 'pixel', (800,600))
```
the alert looks good on my current 1600x900 window and, if I resize the window dynamically (or maximize it), it gets repositioned along with the whole background. It works reasonably well even if I shrink the window quite a lot.